hera_mc/autocorrelations.py

```diff
@@ -132,3 +133,70 @@ def create(cls, time, antenna_number, antenna_feed_pol, measurement_type, value)
             measurement_type=measurement_type,
             value=value,
         )
+
+
+class HeraAutoSpectrum(MCDeclarativeBase):
+    """
+    Definition of hera_auto_spectrum table.
+
+    Attributes
+    ----------
+    time : BigInteger Column
+        The time in GPS seconds of the observation, floored as an int. Part of primary_key
+    antenna_number : Integer Column
+        Antenna number. Part of primary_key.
+    antenna_feed_pol : String Column
+        Feed polarization, either 'e' or 'n'. Part of primary_key.
+    spectrum : Float Columnn
```

Comment:
assumption made here about channel/frequency mapping.  I'm fine with this if you are.

Reply:
I'm not sure what you're getting at. That the frequencies aren't in their own column? That would be pretty redundant because they'd be the same for all rows.

Reply:
I mostly agree with bryna here, but we have done 8k and 16k channel autos before (as a test if I remember but we still did it). But overall it seems like a bit of wasted space to save it all the time. Even then we don't spit out the frequencies in redis anywhere, I'm pretty sure we'd always have to reconstruct them using a linspace and an average (which is what we'd ask a user to do anyway). We could always just write down the algorithm somewhere convenient for anyone who is interested.

Reply:
we could add a column for frequency resolution if that helped. I don't actually know where to find that in redis...

Reply:
I added something to the table definition to describe how to get the frequencies. The basic answer is that the bandwidth (0 -> 250 MHz) is fixed, so the frequencies can be computed from that and from the number of entries in the spectrum.